packages/react/__tests__/__snapshots__/exports.ts.snap

```diff
@@ -2156,9 +2156,6 @@ Object {
       "boxShadow": Object {
         "type": "string",
       },
-      "children": Object {
```

Comment:
We'll probably need to validate that this doesn't break studio. I'm not 100% sure if children is exposed in Studio or not.